### DISCUSSION TITLE
tlg-catalog/stable/tables/pharmacokinetic/pkct01

### DISCUSSION THREAD

**Discussion by giscus** [insightsengineering/tlg-catalog#48] [General]
# tlg-catalog/stable/tables/pharmacokinetic/pkct01

Summary Concentration Table

https://insightsengineering.github.io/tlg-catalog/stable/tables/pharmacokinetic/pkct01.html

**Comment by legrasv** [comment]
split_rows_by(
    var = "NRELTM1",
    split_label = "Norminal Time from First Dose",
    label_pos = "topleft"
  )

I would like to use split_label = 'Nominal Time \nfrom First Dose (hr)' but the "\n" is not recognized, would there be a way to fix that?

**Reply by Melkiades** [reply]
@gmbecker? I think wetook this out when dealing with text wrapping but probably it can be reintroduced. Could you try the wrapping for this? `toString(tbl, widths = <vector_of_column_widths>)`. Use `propose_column_widths()` to see what are the standard values.

**Reply by legrasv** [reply]
Fyi, in the listings, the "\n" is not recognized either.

**Reply by Melkiades** [reply]
Now it works with recent updates ;)

**Comment by KOBANAK** [comment]
Bug: analyze_vars_in_cols fails when combining mean_ci with max
I have encountered an issue where analyze_vars_in_cols works as expected when .stats includes mean_ci, but throws an error if max is appended to the .stats list.

In the first case, the table builds correctly:

```r
# This works correctly
lyt <- lyt_rows %>%
  analyze_vars_in_cols(
    vars = c("AVAL", "AVALCAT1", rep("AVAL", 8)),
    .stats = c("n", "n_blq", "mean", "sd", "cv", "geom_mean", "geom_cv", "median", "min", "mean_ci"),
    .formats = c(
      n = "xx.", n_blq = "xx.", mean = format_sigfig(3), sd = format_sigfig(3), cv = "xx.x", 
      median = format_sigfig(3), geom_mean = format_sigfig(3), geom_cv = "xx.x", min = format_sigfig(3)
    ),
    .labels = c(
      n = "n", n_blq = "Number\nof\nLTRs/BLQs", mean = "Mean", sd = "SD", cv = "CV (%) Mean",
      geom_mean = "Geometric Mean", geom_cv = "CV % Geometric Mean", median = "Median", min = "Minimum"
    ),
    na_str = "NE"
  )

result <- build_table(lyt, df = adpc_1, alt_counts_df = adsl) %>% prune_table()
```
However, adding "max" and updating the variable count causes a failure:
```r
# This throws an error
lyt <- lyt_rows %>%
  analyze_vars_in_cols(
    vars = c("AVAL", "AVALCAT1", rep("AVAL", 9)),
    .stats = c("n", "n_blq", "mean", "sd", "cv", "geom_mean", "geom_cv", "median", "min", "mean_ci", "max"),
    .formats = c(
      n = "xx.", n_blq = "xx.", mean = format_sigfig(3), sd = format_sigfig(3), cv = "xx.x", 
      median = format_sigfig(3), geom_mean = format_sigfig(3), geom_cv = "xx.x", 
      min = format_sigfig(3), max = format_sigfig(3)
    ),
    .labels = c(
      n = "n", n_blq = "Number\nof\nLTRs/BLQs", mean = "Mean", sd = "SD", cv = "CV (%) Mean",
      geom_mean = "Geometric Mean", geom_cv = "CV % Geometric Mean", median = "Median", 
      min = "Minimum", max = "Maximum"
    ),
    na_str = "NE"
  )
result <- build_table(lyt, df = adpc_1, alt_counts_df = adsl) %>% prune_table()
>Error: Error applying analysis function (var - NA): missing value where TRUE/FALSE needed
	occured at (row) path: ACTARM[A: Drug X]->VISIT[Day 1]->NOMTPT[0 / Predose]
```
Any advice on how to resolve this error so I can include all statistics would be greatly appreciated. Thanks for your help!

**Reply by llrs-roche** [reply]
The best way is to get this fixed is to fill a bug report on the package that produces it: https://github.com/insightsengineering/rtables/issues (pinging @shajoezhu too as the maintainer of rtables)
